build.sh

```diff
@@ -17,6 +17,8 @@ BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
 mkdir -p ${BUILD_DIR}
 
 [[ -n "${CCPP_SUITES:-""}" ]] && CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES}"
+[[ -n "${MAPL_ROOT:-""}" ]] && CMAKE_FLAGS+=" -DCMAKE_MODULE_PATH=${MAPL_ROOT}/share/MAPL/cmake"
+CMAKE_FLAGS+=" -DNETCDF_DIR=${NETCDF}"
```

Comment:
Is setting `-DNETCDF_DIR` needed for gocart?

Reply:
Not at all. This is a UFS legacy setting. Would you like me to remove it?

Reply:
If it's not needed remove it.

Reply:
Done in 0fab7d804b2452c5cb8f569742a8766e9db10d31